.github/workflows/release.yml

```diff
@@ -0,0 +1,95 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Release
+
+jobs:
+  build:
+    name: Build and release
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: "on"
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v2-beta
+        with:
+          go-version: 1.14.x
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build All
+        shell: bash
+        env:
+          SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+        run: |
+          SENTRY_DSN=$SENTRY_DSN make buildAll
```

Comment:
Are you sure you can build all binaries on ubuntu? 
I think you need to run in either in docker or on appropriate os machine

Reply:
Yep
Tested locally and builds for correct os

Reply:
@mpetrun5 ain't binaries suppose to have appropriate extensions. For example .exe on windows?

Reply:
That is a mistake
I overwritten .exe on build